.github/workflows/ci_test.yml

```diff
@@ -26,9 +26,10 @@ jobs:
       - name: Run python tests
         run: |
           pip install -r requirements.txt
+          version=$(python -m armory --version)
           bash docker/build-dev-minimal.sh
-          pip install -r test-requirements.txt
-          pytest -s --disable-warnings
+          docker run twosixarmory/tf1:${version} bash -c \
+          "cd /armory_dev; pytest -s --disable-warnings --ignore=tests/test_docker_management.py"
```

Comment:
nit: 
```suggestion
          docker run -w /armory_dev twosixarmory/tf1:${version} bash \
          pytest -s --disable-warnings --ignore=tests/test_docker_management.py
```

Reply:
Is the "-w" run from a specific directory? I looked for a flag that did that, but the docker docs can be hard to parse.

Reply:
Yup, agree so many docs its confusing:
https://docs.docker.com/engine/reference/commandline/run/#options

Reply:
Modified.